bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,65 @@
+import random
+
+from cowsay import char_names, get_output_string
+from discord import Embed
+from discord.ext import commands
+
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+
+class Cowsay(commands.Cog):
+    """Cog for the cowsay command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(
+        aliases=["cow"],
+        help=f"""
+        Generates a cowsay string and sends it. The available characters for cowsay are:
+
+        {', '.join(char_names)}"""
+    )
+    async def cowsay(self, ctx: commands.Context, character: str = "Cow", *, text: str = None) -> None:
+        """
+        Generates some cowsay ASCII art and sends it in Discord.
+
+        There are multiple character options to choose from. Here is a list:
+        """
```

Comment:
This can be shortened and not be repetitive of what is said in the `help=` parameter of `discord.exts.commands()`.

Reply:
And also note, this doc-string is not send or visible to a user on discord, it is only there for a contributor to see.

Reply:
Alright

Reply:
@ToxicKidz ive resolved the changes you requested